handbook/community.md

```diff
@@ -82,6 +82,37 @@ The images used in the docs live in `docs/images/`. Note that you must provide t
 
 > Note that the instructions above also apply to adding images in the Fleet handbook.
 
+### Adding a mermaid diagram to the Fleet Docs
+
+The Fleet Docs support diagrams written in the mermaid.js. Take a look at the [mermaid docs](https://mermaid-js.github.io/mermaid/#/README) to learn about the syntax language and what types of diagrams you can display with mermaid.
```

Comment:
```suggestion
The Fleet Docs support diagrams that are written in mermaid.js syntax. Take a look at the [Mermaid docs](https://mermaid-js.github.io/mermaid/#/README) to learn about the syntax language and what types of diagrams you can display.
```

Reply:
I made some edits to reduce the amount of time we are referring to [mermaids](https://www.youtube.com/watch?v=AHGDVUqY9QI).